dist/newrelic/newrelic.yml

```diff
@@ -0,0 +1,302 @@
+# This file configures the New Relic Agent.  New Relic monitors
```

Comment:
We already have a newrelic.yml https://github.com/Sage-Bionetworks/BridgePF/blob/develop/newrelic.yml

Why do we need a second one, and why is this different from the current one?

Also, iirc, a lot of the settings shouldn't be specified in the newrelic.yml file, because a lot of the settings can be configured directly through the New Relic web UI.